src/client/client4.js

```diff
@@ -1095,14 +1302,15 @@ export default class Client4 {
     };
 
     logClientError = async (message, level = 'ERROR') => {
-        const body = {
-            message,
-            level
-        };
+        if (!this.enableLogging) {
```

Comment:
You're logging the RN errors to the server? It will work if it's in developer mode